### PR TITLE
Import CookieHelper library from NPM (Fixes #7)

### LINKED ISSUE #7: Import cookie helper lib from NPM
Now that the cookie helper lib is [published to NPM](https://www.npmjs.com/package/@mozmeao/cookie-helper), we should update traffic cop to import this as a dependency.

We should also document how Traffic Cop depends on the cookie helper library (expecting it as a global etc).

### PR DESCRIPTION
**Pull request by alexgibson**
- Adds a webpack config for the demo pages to load `@mozmeao/cookie-helper` from NPM.
- Updates documentation to include info about the cookie helper dependency.
- Removes duplicated cookie helper lib source code from the repo.

To test:

1. `cd demo`
2. `npm install`
3. `npm start`
4. Open http://localhost:3030/
5. Ensure the traffic cop example still work correctly.

### CODE REVIEW

File: demo/webpack.config.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    },
+    performance: {
+        hints: 'warning'
+    },
+    optimization: {
+        minimize: false
+    },
+    plugins: [
+        // clean out demo/libs/ before building
+        new CleanWebpackPlugin()
```
Comment (nathan-barrett) [inline demo/webpack.config.js:18]: **Question**: When I was setting these libraries up with Webpack I noticed that there is a property you can pass to output called `clean` but I haven't looked up the pros/cons of each. Might be worth looking into if we could replace with the property to remove that as a dependency for these packages (and bedrock too)

File: documentation.md [hunk 2]
```diff
@@ -178,10 +178,25 @@ Mozilla.Cookies.removeItem('mozilla-traffic-cop-original-referrer');
 
 Traffic Cop requires three JavaScript files:
 
-1. `mozilla-cookie-helper.js` (included in the `src` directory)
-2. `mozilla-traffic-cop.js`
+1. `@mozmeao/cookie-helper` (install via [NPM](https://www.npmjs.com/package/@mozmeao/cookie-helper))
+2. `@mozmeao/trafficcop` (install via [NPM](https://www.npmjs.com/package/@mozmeao/trafficcop))
```
Comment (nathan-barrett) [inline documentation.md:182]: **Question:** When is best to tell users to install the cookie helper on their own via NPM vs having CookieHelper as a dependency of TrafficCop?

**Review by nathan-barrett — APPROVED** [review]
R+wc just some questions otherwise a good improvement 🧄

File: demo/webpack.config.js (hunk 1 above)
Comment (alexgibson) [reply demo/webpack.config.js:18]: Oh nice, let’s look into this and see if we can simplify a few projects using it 🙂

File: documentation.md (hunk 2 above)
Comment (alexgibson) [reply documentation.md:182]: Shamelessly I think for now we need to keep the dependencies as installed separately, mostly for bedrock. As the cookie helper lib is used for many different things there. It needs to exist in an entirely separate bundle to traffic cop.

I’m not sure if anyone outside our own team using Traffic Cop. Perhaps if we see that change, we could figure something better out.

**Pull request merged**